README.md

```diff
@@ -1,8 +1,44 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Binsar Hutapea . Jarvis Consulting
 
+I am very enthusiastic about venturing into the world of Software Development to seek the challenges of utilizing it in the industry to solve real-life problems. I have prior industry experience as an application developer, where I was deeply involved in developing, testing, and deploying multiple web-based applications for multiple corporations, where I used Java and Typescript within Spring and Angular frameworks for performing programming tasks, MySQL and Oracle for interacting with the databases, and Selenium for performing automated testing.
```

Comment:
Intro is good but just a little short. Maybe include a line or two at the end about what skills/projects you are currently developing, or what you focused on during your education before that.

Reply:
Thank you for the feedback. I have added a few lines to the intro.